app/controllers/chat_channels_controller.rb

```diff
@@ -181,7 +181,7 @@ def channel_info
 
   def create_channel
     chat_channel_params = params[:chat_channel]
-    chat_channel_name = chat_channel_params[:channel_name].split(" ").join("-")
+    chat_channel_name = chat_channel_params[:channel_name].split.join("-")
```

Comment:
you'll notice a bunch of these throught the PR. `split(" ")` and `.split` are the same thing. Splitting over space char is the default behavior

Reply:
:heart_eyes: